src/workerd/api/node/node-version.h

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2023 Cloudflare, Inc.
+// Licensed under the Apache 2.0 license found in the LICENSE file or at:
+//     https://opensource.org/licenses/Apache-2.0
+#pragma once
+
+#include <kj/string.h>
+
+namespace workerd::api::node {
+
+// Node.js version supported by the platform.
+// This represents the most current Node.js version supported by the platform
+// and will change as Node.js release updates ship.
```

Comment:
It's likely worth expanding on the fact that this is an approximation. We do not fully support this specific version (as there are APIs we know are missing) and we may support features that shipped after this version.

Reply:
Added some clarification.